### ISSUE TITLE
Indicate where get nativeView fails

### ISSUE THREAD

**Issue by vallemar** [nativescript-vue/nativescript-vue#796]
### What problem does this feature solve?
It is very annoying and difficult to debug where it is giving you a TypeError exception: Cannot read property 'nativeView' of undefined. Many times it takes longer to find where it is wrong than to correct it.

### What does the proposed API look like?
In the exception information you should put the line and the file where the error is giving.

**Comment by rigor789** [comment]
Do you have an example where you get this TypeError?

**Comment by vallemar** [comment]
Yes, [here is an example](https://play.nativescript.org/?template=play-vue&id=vvagvF). This seems very basic but when you have a few frames loaded in the same view it is an odyssey to look for the error.

**Comment by vallemar** [comment]
In general I would say that the traces are quite complicated to follow since it is not usually indicated where the error comes from, it usually says the internal error but not where your code is testing it. I would understand that this improvement applies to Nativescript more than to nativescript-vue, I have not used another nativescript so I have created it here.

**Comment by rigor789** [comment]
So the general TypeError of `TypeError: Cannot read property 'xxxxx' of undefined` is not something we can really hook into/change, but having better traces that point to the actual line in your code would be useful. In fact, it supposed to work in most cases, but doesn't seem to in this one where the activity crashes on creation.

The stack trace does have the file for me, just not the line btw:
```
JS: [Vue warn]: Error in created hook: "TypeError: Cannot read property 'nativeView' of undefined"
JS:
JS: found in
JS:
JS: ---> <Test> at components/Test.vue
JS:        <Frame>
JS:          <Root>
System.err: An uncaught Exception occurred on "main" thread.
System.err: Unable to start activity ComponentInfo{org.nativescript.myVueTSApp/com.tns.NativeScriptActivity}: com.tns.NativeScriptException: Calling js method onCreate failed
System.err: TypeError: Cannot read property 'nativeView' of undefined
...
```

**Comment by vallemar** [comment]
I had not realized that some exceptions do report the file and the line (these must be the one that occurs the most but I had not realized it). Personally I think that the ones I have found the most are those that do not follow the trace, that is why I have opened this topic. Something interesting would be a link that takes you to the file->line as java (I don't know if it can but it would be very comfortable).

What we can do if you see it correctly is to resolve these exceptions and if later we continue to see exceptions without a trace, we open another topic until we polish this more.
